plugin/pulpcore/plugin/download/asyncio/exceptions.py

```diff
@@ -0,0 +1,34 @@
+from gettext import gettext as _
+
+from pulpcore.exceptions import PulpException
+
+
+class DownloaderValidationError(PulpException):
+    """
+    A base class for all Download Validation Errors.
+    """
+    pass
+
+
+class DigestValidationError(DownloaderValidationError):
+    """
+    Raised when an Artifact fails to validate a digest checksum.
+    """
+
+    def __init__(self):
+        super().__init__("PLP0003")
+
+    def __str__(self):
+        return _("An Artifact failed validation due to checksum.")
```

Comment:
What if the file being downloaded w/ validation is not an artifact?  Misleading?  Also, I think including the url would be helpful.

Reply:
This is a leftover from a name change. Next push will update with s/An Artifact/A file/

We can't include any useful data (it sucks) until we resolve https://pulp.plan.io/issues/2988